app/lpc55xpresso/stage0.toml

```diff
@@ -22,3 +22,7 @@ start = true
 [signing]
 priv-key = "../../support/fake_certs/fake_private_key.pem"
 root-cert = "../../support/fake_certs/fake_certificate.der.crt"
+enable-dice = true
+dice-inc-nxp-cfg = false
+dice-cust-cfg = false
+dice-inc-sec-epoch = false
```

Comment:
Does it make sense to put this on other targets? e.g. gemini-bu-rot. Those targets won't have TrustZone but secureboot and DICE should still work.

Reply:
Good point. I'll take a pass over the app configs with board set to 'lpcxpresso55s*' and make similar changes.

Reply:
fixed by last push